src/cache/gcs.rs

```diff
@@ -572,12 +572,15 @@ impl Storage for GCSCache {
         let data = entry.finish()?;
 
         let bucket = self.bucket.clone();
-        let _response = bucket
+        let response = bucket
             .put(&key, data, &self.credential_provider)
             .await
-            .context("failed to put cache entry in GCS")?;
+            .context("failed to put cache entry in GCS");
 
-        Ok(start.elapsed())
+        match(response) {
```

Comment:
`.map(move |_| start.elapsed())` is more idiomatic instead of a match

Reply:
Actually, the original code was fine here. `let response = ` is just too much, since the value really is `()`.